buildSrc/src/main/java/com/uber/okbuck/core/manager/JetifierManager.java

```diff
@@ -0,0 +1,107 @@
+package com.uber.okbuck.core.manager;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.sun.istack.Nullable;
+import com.uber.okbuck.OkBuckGradlePlugin;
+import com.uber.okbuck.composer.base.BuckRuleComposer;
+import com.uber.okbuck.core.dependency.DependencyCache;
+import com.uber.okbuck.core.model.base.RuleType;
+import com.uber.okbuck.core.util.FileUtil;
+import com.uber.okbuck.core.util.ProjectUtil;
+import com.uber.okbuck.template.core.Rule;
+import com.uber.okbuck.template.java.Prebuilt;
+import com.uber.okbuck.template.jvm.JvmBinaryRule;
+
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.dsl.DependencyHandler;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Set;
+
+public final class JetifierManager {
+
+    private static final String JETIFIER_LOCATION =
+            OkBuckGradlePlugin.WORKSPACE_PATH + "/jetifier";
+    private static final String JETIFIER_BUCK_FILE = JETIFIER_LOCATION + "/BUCK";
+    private static final String JETIFIER_DEPS_CONFIG = "okbuck_jetifier_deps";
+    private static final String JETIFIER_CACHE_LOCATION = "com/android/tools/build/jetifier";
+    private static final String JETIFIER_GROUP = JETIFIER_CACHE_LOCATION
+            .replace("/", ".");
+    private static final String JETIFIER_CLI_CLASS = "com.android.tools.build.jetifier.standalone.Main";
+    private static final String JETIFIER_BINARY_RULE_NAME = "okbuck_jetifier";
+    private static final String JETIFIER_VERSION = "1.0.0-beta02";
+
+    private static final ImmutableList<String> JETIFIER_MODULES =
+            ImmutableList.of(
+                    "jetifier-core",
+                    "jetifier-processor"
+            );
+    private static final ImmutableList<String> INTERNAL_MODULES =
+            ImmutableList.of(
+                    "commons-cli.jar",
+                    "jetifier-standalone.jar"
+            );
```

Comment:
these jars doesn't come coupled with jetifier?

Reply:
According to the maintainers, they're not supposed to. Why do we need them?

Reply:
We run jetifier as a cli so we need the standalone jar to call into jetifier-core